src/app/core/services/auth.service.ts

```diff
@@ -23,6 +23,7 @@ export enum AuthState { 'NotAuthenticated', 'AwaitingAuthentication', 'Authentic
 export class AuthService {
   authStateSource = new BehaviorSubject(AuthState.NotAuthenticated);
   currentAuthState = this.authStateSource.asObservable();
+  private userObj;
```

Comment:
Do you need userObj? If not, you can delete it.

Reply:
My bad, I have removed the object